src/components/Data/Documents/Common/CreateOrUpdate.vue

```diff
@@ -86,7 +86,7 @@
           <div class="col s7 m8 l9" v-if="error">
             <div class="card error red-color">
               <i class="fa fa-times dismiss-error" @click="dismissError()"></i>
-              <p v-html="error"></p>
+              <p class="white-text" v-html="error"></p>
```

Comment:
You add a `white-text` class here, and the css rule apply to `.error-text`. is it normal? 
(the same for each error card you modified)